blueprints/ember-frost-css-core/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.0.2'}
```

Comment:
@vesper2000 @sglanzer Let me see if I understand this properly. I *think* I have my head wrapped around it, and I *think* I have a suggestion on how to make it a little better to work with.

Let's first assume that `ember-frost-css-core` **DOES NOT** list `ember-frost-theme` in it's `package.json` If I understand correctly, a consumer can still `ember install ember-frost-css-core` and properly receive `ember-frost-theme` as well, thanks to the lines above. However, if someone wants to work on `ember-frost-css-core` itself, the `npm install` on a fresh clone will be missing `ember-frost-theme`. There are two solutions for that. The first is to put the `ember-frost-theme` in the `package.json` for `ember-frost-css-core` under `dependencies`. The second is to put the `ember-frost-theme` in the `package.json`. for `ember-frost-css-core` under `devDependencies`.  Both of these will result in a fresh `npm install` of a cloned repo installing `ember-frost-theme`, but the former has the side-effect that if someone does an `npm install ember-frost-css-core` instead of an `ember install ember-frost-css-core` then `ember-frost-theme` will be installed in the consumer as well (although depending on how it's done, it may be at `node_modules/ember-frost-css-core/node_modules/ember-frost-theme` instead of `node_modules/ember-frost-theme`. It will also **not** execute the blueprints for `ember-frost-theme`, which could break things in hard to debug ways. If we put it in `devDependencies` instead, when the consumer mistakenly uses `npm install` instead of `ember install` to install the addon, the dependencies will be missing altogether, which, to me is a clearer indication that something was installed incorrectly. 

@sglanzer @vesper2000 @sandersky Thoughts?